### DISCUSSION TITLE
Enhancement: Script to Set Balance

### DISCUSSION THREAD

**Discussion by KATADA-Kazuyuki** [danny-avila/LibreChat#4320] [Feature Requests & Suggestions]
### What features would you like to see added?

Currently, the add-balance.js script is implemented to add tokens. I would like it to support both add and set operations.
The set operation should overwrite the current token balance regardless of the existing amount.

SAMPLE: npm run add-balance <email> <amount> [add/set]

### More details

set 2000 token  : npm run add-balance dummy@dummy.com 2000 **set**
add 2000 token  : npm run add-balance dummy@dummy.com 2000

### Which components are impacted by your request?

Other

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Merged this in: https://github.com/danny-avila/LibreChat/pull/4506
